### PR TITLE
Adding multi disk selection for supported templates

### PR DESCRIPTION
**Pull request by davidjumani**
Adding support for non deploy as is templates which have multiple data disks

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/views/compute/DeployVM.vue [hunk 1]
```diff
@@ -1484,6 +1524,12 @@ export default {
           deployVmData.bootintosetup = values.bootintosetup
         }
 
+        // if (deployVmData) {
```
Comment (nvazquez) [inline src/views/compute/DeployVM.vue:1527]: Remove me :)

**Review by nvazquez — COMMENTED** [review]
Thanks @davidjumani this is looking very good. Just left a minor comment

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by andrijapanicsb** [comment]
LGTM

**Pull request merged**